lib/cash-out/cash-out-tx.js

```diff
@@ -44,7 +44,11 @@ function selfPost (tx, pi) {
 }
 
 function post (tx, pi, fromClient = true) {
-  return db.tx(cashOutAtomic.atomic(tx, pi, fromClient))
+  const TransactionMode = pgp.txMode.TransactionMode
+  const isolationLevel = pgp.txMode.isolationLevel
+  const mode = new TransactionMode({ tiLevel: isolationLevel.serializable })
+
+  return db.tx({ mode }, cashOutAtomic.atomic(tx, pi, fromClient))
```

Comment:
Maybe we can move the tx to inside cashOutAtomic. That way the configuration stays in place and we're actually leaving the file named atomic as atomic, same thing for cashIn